lib/3scale_toolbox/commands/service_command/service_show.rb

```diff
@@ -0,0 +1,53 @@
+module ThreeScaleToolbox
+  module Commands
+    module ServiceCommand
+      class ServiceShowSubcommand < Cri::CommandRunner
+        include ThreeScaleToolbox::Command
+
+        def self.command
+          Cri::Command.define do
+            name        'show'
+            usage       'show'
+            summary     'service show'
+            description "Show the information of a service"
+            runner ServiceShowSubcommand
+
+            param   :source
+            param   :service_id
+          end
+        end
+
+        SERVICE_FIELDS_TO_SHOW = %w[
```

Comment:
I've seen a very similar array was defined in https://github.com/3scale/3scale_toolbox/blob/7a5e540e2919bd069ba416f27139daa654211f47/lib/3scale_toolbox/entities/service.rb#L4
although that is used for another purposes and some of the fields are omitted.
Do you think we should move that variable there? split arrays or something?
It seems very specific of the specific output of the command although they are field names contained in the result.

Reply:
Maybe in `ThreeScaleToolbox::Commands::ServiceCommand` namespace??